objects/rct1/footpath_surface/rct1.footpath_surface.road/object.json

```diff
@@ -0,0 +1,43 @@
+{
+    "id": "rct1.footpath_surface.road",
+    "authors": [
+        "Chris Sawyer",
+        "Simon Foster"
+    ],
+    "version": "1.0",
+    "sourceGame": [
+        "rct1",
+        "rct2"
+    ],
+    "objectType": "footpath_surface",
+    "properties": {
+        "editorOnly": true,
+        "noSlopeRailings": true
```

Comment:
Why not just adding this property to the existing rct2 road?

Reply:
That would affect how existing parks look, and that always attracts complaints. So it has basically become policy to make sure all object changes are fully backwards compatible.

The way I did it, both RCT1 and RCT2 road will be drawn correctly, but only one will show up in the Object Selection, so it doesn’t clutter things up.

Reply:
Fair enough, I guess

Reply:
Thanks!